tools/serverpod_cli/lib/src/generator/dart/library_generators/model_library_generator.dart

```diff
@@ -947,6 +947,18 @@ class SerializableModelLibraryGenerator {
         return refer(field.type.className)
             .property('parse')
             .call([CodeExpression(Code("'$defaultValue'"))]).code;
+      case DefaultValueAllowedType.bool:
+        return switch (defaultValue) {
+          defaultBooleanTrue => literalBool(true).code,
+          defaultBooleanFalse => literalBool(false).code,
+          _ => null,
+        };
+      case DefaultValueAllowedType.int:
+        return literalNum(int.parse(defaultValue)).code;
+      case DefaultValueAllowedType.double:
+        return literalNum(double.parse(defaultValue)).code;
+      case DefaultValueAllowedType.string:
+        return Code(defaultValue);
```

Comment:
Suggestion: Use `literalString`.

Any reason for not using `literalString` here?

Reply:
Yes, because it will be "\'This is a string\'", it will have double quotes.